src/Components/test/E2ETest/Tests/JSRootComponentsTest.cs

```diff
@@ -36,6 +36,7 @@ protected override void InitializeAsyncCore()
     [InlineData(false, true)]
     [InlineData(true, false)]
     [InlineData(true, true)]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/38613")]
```

Comment:
These tests are still running/failing despite being quarantined. Any thoughts an why this may be the case @dotnet/aspnet-build? Is there additional config we may need to do for `aspnetcore-components-e2e`?

<img width="1071" alt="Screen Shot 2021-11-29 at 8 26 34 AM" src="https://user-images.githubusercontent.com/14852843/143905273-d7ea41d4-2aee-4eae-8ae6-7ae9e8c5f99d.png">

Reply:
I think it's because these tests run with `dotnet test` and not xunit, so the `TestRunnerAdditionalArguments` don't get used:

https://github.com/dotnet/aspnetcore/blob/06483b765b7c6a5657784c817b4feeae000c1fe8/eng/QuarantinedTests.BeforeArcade.props#L13

I'd suggest trying a `dotnet test` filter based on `traits` as described here: https://docs.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=mstest

https://github.com/dotnet/aspnetcore/blob/06483b765b7c6a5657784c817b4feeae000c1fe8/.azure/pipelines/components-e2e-tests.yml#L47-L48

Reply:
Thanks Will, trying it out here:

https://github.com/dotnet/aspnetcore/pull/38693/commits/ded632e3fe7b08e49eae209e86f611c6b59e6c43

Reply:
I thought @SteveSandersonMS's _intention_ was to cover both quarantined and unquarantined tests. The unfortunate part here is the lack of distinction in the results between the two.